client/src/main/scala/com/dwolla/cloudflare/AccountsClient.scala

```diff
@@ -29,10 +30,20 @@ trait AccountsClient[F[_]] {
   def addMember(accountId: AccountId, emailAddress: String, roleIds: List[String]): Stream[F, AccountMember]
   def updateMember(accountId: AccountId, accountMember: AccountMember): Stream[F, AccountMember]
   def removeMember(accountId: AccountId, accountMemberId: String): Stream[F, AccountMemberId]
+
+  def getByUri(uri: String): Stream[F, Account] = parseUri(uri).fold(Stream.empty.covaryAll[F, Account])(getById)
+
+  def parseUri(uri: String): Option[AccountId] = uri match {
+    case AccountsClient.uriRegex(accountId) ⇒ Option(tagAccountId(accountId))
+    case _ ⇒ None
+  }
+
 }
 
 object AccountsClient {
   def apply[F[_] : Sync](executor: StreamingCloudflareApiExecutor[F]): AccountsClient[F] = new AccountsClientImpl[F](executor)
+
+  val uriRegex: Regex = """https://api.cloudflare.com/client/v4/accounts/(.+?)""".r
```

Comment:
Do we need a regex to parse the account member id out also? I believe thats what is happening in the cloudflare-lambda.

Reply:
I think that's a good idea. I tried to stub out an implementation of `parseUri` that handled both URI forms but it got pretty unwieldy. I think it will work better to move account member methods to a separate `AccountMembersClient`? What do you think?

Reply:
e.g. 

```scala
def parseUri(uri: String): Option[Either[(AccountId, AccountMemberId), AccountId]]
```

which then propagates out to `getByUri`, etc.

Reply:
Yeah, I think it would work to move those methods out.